requirements-btchip.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/LedgerHQ/btchip-python.git@fe292218e1c9c4939071650c729770bb6b34aa1b
```

Comment:
Use tag instead of commit : git+https://github.com/LedgerHQ/btchip-python.git@v0.1.16 ?